doc/services/serialization/nanopb.rst

```diff
@@ -45,6 +45,7 @@ Additionally, Nanopb is an optional module and needs to be added explicitly to t
 
    west config manifest.project-filter -- +nanopb
    west update
+   west packages pip --install
```

Comment:
note to self: might be worth having a Sphinx directive similar to `zephyr-app-commands` but for optional module provisioning

Reply:
Create an issue?